### DISCUSSION TITLE
Unable to load local tiles

### DISCUSSION THREAD

**Discussion by devWaleed** [react-native-maps/react-native-maps#4629] [Q&A]
I have saved the following file in my physical device storage using RNFetchBlob: https://stamen-tiles.a.ssl.fastly.net/toner/10/65/25.png 

I am able to verify that the file exist physical device using the following code:

```
const downloadedImage = await RNFetchBlob.fs.exists('file://' + image.path);

console.log('downloaded image', downloadedImage); // returns true
```

I am trying to show the image as one tile on map like this:

```
<MapView
  zoomControlEnabled
  showsScale={true}
  mapType="standard"
  provider={PROVIDER_GOOGLE}
  style={styles.map}
  minZoomLevel={0}
  maxZoomLevel={11}>
  <LocalTile
    pathTemplate={
      'file:///storage/emulated/0/Download/tiles/{z}-{x}-{y}.png'
    }
    tileSize={256}
    zIndex={999999}
  />

  <Marker
    focusable
    coordinate={{
      latitude: coordinates.x,
      longitude: coordinates.y,
    }}
  />
</MapView>
```

This doesnt draw on map, for further proof that image exist and can be read, I put the same path as Image source and it showed up correctly:

```
 <Image
        style={{
          width: 100,
          height: 100,
        }}
        source={{
          uri: 'file:///storage/emulated/0/Download/tiles/10-25-66.png',
        }}
      />
```

I don't understand what I am doing wrong here. For now my requirement is to just replace a single tile over an area with the image, later I will have set of multiple tiles. But I am not able to show this single tile.

**Comment by aroshanzamir** [comment]
> I have saved the following file in my physical device storage using RNFetchBlob: https://stamen-tiles.a.ssl.fastly.net/toner/10/65/25.png
> 
> I am able to verify that the file exist physical device using the following code:
> 
> ```
> const downloadedImage = await RNFetchBlob.fs.exists('file://' + image.path);
> 
> console.log('downloaded image', downloadedImage); // returns true
> ```
> 
> I am trying to show the image as one tile on map like this:
> 
> ```
> <MapView
>   zoomControlEnabled
>   showsScale={true}
>   mapType="standard"
>   provider={PROVIDER_GOOGLE}
>   style={styles.map}
>   minZoomLevel={0}
>   maxZoomLevel={11}>
>   <LocalTile
>     pathTemplate={
>       'file:///storage/emulated/0/Download/tiles/{z}-{x}-{y}.png'
>     }
>     tileSize={256}
>     zIndex={999999}
>   />
> 
>   <Marker
>     focusable
>     coordinate={{
>       latitude: coordinates.x,
>       longitude: coordinates.y,
>     }}
>   />
> </MapView>
> ```
> 
> This doesnt draw on map, for further proof that image exist and can be read, I put the same path as Image source and it showed up correctly:
> 
> ```
>  <Image
>         style={{
>           width: 100,
>           height: 100,
>         }}
>         source={{
>           uri: 'file:///storage/emulated/0/Download/tiles/10-25-66.png',
>         }}
>       />
> ```
> 
> I don't understand what I am doing wrong here. For now my requirement is to just replace a single tile over an area with the image, later I will have set of multiple tiles. But I am not able to show this single tile.

I have the same problem. did you find any solution?

**Reply by Cdingram** [reply]
Same. If you use file:// try using urltile instead of localtile, or try without the file:// part with localtile. I've seen that done other places however neither work for me currently.
